src/js/pages/task-details/TaskDetail.js

```diff
@@ -261,12 +301,17 @@ class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
     }
 
     return (
-      <RouteHandler
-        directory={directory}
-        onOpenLogClick={this.handleOpenLogClick.bind(this)}
-        selectedLogFile={selectedLogFile}
-        service={this.getService()}
-        task={task} />
+      <div className="flex-container-col flex-grow flex-shrink">
```

Comment:
These flex classes have changed a bit:
`flex-container-col` === `flex flex-direction-top-to-bottom` (two classes)
`flex-grow` === `flex-item-grow-1`
`flex-shrink` === `flex-item-shrink-1`
`flex-box` === `flex`